packages/camera/camera_android_camerax/pubspec.yaml

```diff
@@ -31,4 +31,4 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   mockito: 5.4.0
-  pigeon: ^3.2.6
+  pigeon: ^9.1.0
```

Comment:
@stuartmorgan - I cannot find any file specifying the arguments to `flutter pub run pigeon` to use for this package, so I cannot test that the breaking changes in `pigeon` in the last 8 months do not impact this package.

How can I find the command to run?

Reply:
It's probably just `--input pigeons/camerax_library.dart`, but I don't work on that package so I don't know; @camsim99 would be the best person to answer that. (Tooling for keeping Pigeon updated in the repo is on my TODO list, FWIW.)

Any Pigeon updates should be done as separate precursor PRs since they impact production code.

Reply:
`flutter pub run pigeon --input pigeons/camerax_library.dart` is what I use when working on this package

Reply:
> Any Pigeon updates should be done as separate precursor PRs since they impact production code.

https://github.com/flutter/packages/pull/3561

Does this package impact google3? Or does it impact production without being published?

Reply:
> Or does it impact production without being published?

I'm not sure I understand the question. The generated code is production code, which means once it's published it affects clients. And since it affects clients, it needs to be published by repo policy.

Reply:
> I'm not sure I understand the question.

I was thrown off by the `publish_to: none`, but I guess one of the files is generated into some other package that is published?

https://github.com/flutter/packages/blob/e893fdc9617c84b69737edbbdc681501289ed8d4/packages/camera/camera_android_camerax/pubspec.yaml#L5